Assets/Scripts/Object/AbstractObject.cs

```diff
@@ -23,16 +24,23 @@ public void Start() {
                 ? Random.Range(minRotSpeed, maxRotSpeed)
                 : defaultRotationSpeed;
             rotDirection = Vector.GetRandomDirection();
-
             OnSpawn();
         }
 
         public virtual void OnSpawn() { }
-
+         
         public void Update() {
             float moveDelta = moveSpeed * Time.deltaTime;
+            moveSpeed = moveSpeed + 0.01F; // A bit of acceleration
             Vector3 forwardVector = (targetPosition - transform.position).normalized;
             transform.position += moveDelta * forwardVector;
+            // Destroy object within BH
+            if(transform.position.magnitude < BH_radius){
```

Comment:
![ask](https://img.shields.io/badge/review-ask-9b71ae.svg) 
Black hole position should be flexible (not fixed at (0, 0, 0))